frontend/src/GameModule/CreateNewGame/components/CreateNewGame.vue

```diff
@@ -58,7 +65,7 @@ const onChangeImageFile = (event: Event) => {
 
 <template>
   <router-link to="/home" class="button">DONE</router-link>
-  <form @submit.prevent="onCreateNewGame">
+  <form @submit.prevent="onCreateNewGame" enctype="multipart/form-data">
```

Comment:
what is enctype="multipart/form-data" for? Why not just set this content type in the request init's header if it is needed?

Reply:
Shoot forgot to remove this. It should already be set in the request header if it needs it, logic is defined in callApi